### PR TITLE
Add WMTS link object to EDC byoc collection metadata

### PR DESCRIPTION
**Pull request by horvatdino**
Closes: https://git.sinergise.com/team-6/openeo-platform/-/issues/154

- Add instance for openEOP EDC collection to SH account
- Add link object to collections .yaml files

### CODE REVIEW

File: collections/sea-ice-index.yaml [hunk 1]
```diff
@@ -44,7 +44,12 @@ Resources:
     Role: processor
     Type: byoc-10549890-13bb-4ec5-8ae0-df387bd0b785
     CollectionId: 10549890-13bb-4ec5-8ae0-df387bd0b785
-    Primary: true  
+    Primary: true
+WMTS:
+  - href: https://creodias.sentinel-hub.com/ogc/wmts/2a0231a2-5c91-4d3b-a911-ff55711aa983
```
Comment (dthiex) [inline collections/sea-ice-index.yaml:49]: I didn't manage to test this (see my question in the Sentinel chat). Did this work for you when you tested it?

**Review by dthiex — CHANGES_REQUESTED** [review]

File: collections/sea-ice-index.yaml [hunk 2]
```diff
@@ -44,7 +44,12 @@ Resources:
     Role: processor
     Type: byoc-10549890-13bb-4ec5-8ae0-df387bd0b785
     CollectionId: 10549890-13bb-4ec5-8ae0-df387bd0b785
-    Primary: true  
+    Primary: true
+WMTS:
```
[the remark this answers is not in the document]
Comment (horvatdino) [reply collections/sea-ice-index.yaml:48]: This was done - now the logo is disabled for all instances.

[the remark this answers is not in the document]
Comment (dthiex) [reply collections/sea-ice-index.yaml:48]: Great, thanks!

File: collections/sea-ice-index.yaml (hunk 1 above)
Comment (horvatdino) [reply collections/sea-ice-index.yaml]: WMTS link object for this collection was removed until it's fixed. See [issue](https://git.sinergise.com/sentinel-core/java/-/issues/2470).

File: collections/sea-ice-index.yaml [hunk 3]
```diff
@@ -44,7 +44,7 @@ Resources:
     Role: processor
     Type: byoc-10549890-13bb-4ec5-8ae0-df387bd0b785
     CollectionId: 10549890-13bb-4ec5-8ae0-df387bd0b785
-    Primary: true  
+    Primary: true
```
Comment (dthiex) [inline collections/sea-ice-index.yaml:47]: You added again some empty lines. Please remove those. Afterwards you can merge this :)

**Review by dthiex — APPROVED** [review]

File: collections/sea-ice-index.yaml (hunk 3 above)
Comment (horvatdino) [reply collections/sea-ice-index.yaml]: Removed. Need approval for merging :)

**Review by dthiex — APPROVED** [review]

**Pull request merged**